pkg/node/node.go

```diff
@@ -99,17 +99,28 @@ func (n *Node) GetNodeIP(ipv6 bool) net.IP {
 func (n *Node) getPrimaryAddress(ipv4 bool) *models.NodeAddressing {
 	v4, v4Type := n.getNodeIP(false)
 	v6, v6Type := n.getNodeIP(true)
+
+	var (
+		ipv4AllocStr = "not defined"
+		ipv6AllocStr = "not defined"
```

Comment:
This will end up getting picked up in `pkg/health/prober.go` and it will attempt to resolve it.

Can we instead set the string to empty, set "Enabled" in the models below to false, then print things like "not defined" on the CLI (`pkg/client/client.go`) side rather than in the API response?